src/app/ui-library/callout.tsx

```diff
@@ -0,0 +1,94 @@
+import { CheckCircle, Info, Warning, WarningCircle } from '@phosphor-icons/react';
+import { createElement } from 'react';
+import styled from 'styled-components';
+import Theme from 'theme';
+import { StyledHeading, StyledP } from './common.styled';
+
+type CalloutVariant = 'info' | 'warning' | 'danger' | 'success';
+
+const backgroundColors = {
+  info: Theme.colors.elevation6_600,
+  warning: Theme.colors.caution_800,
+  success: Theme.colors.success_medium_700,
+  danger: Theme.colors.danger_dark_600,
+};
+const getBackgroundForVariant = (variant: CalloutVariant) => backgroundColors[variant];
+
+const icons = {
+  info: Info,
+  warning: WarningCircle,
+  success: CheckCircle,
+  danger: Warning,
+};
+const getIconForVariant = (variant: CalloutVariant) =>
+  createElement(icons[variant], {
+    weight: 'fill',
+    size: '24',
+    color: Theme.colors.white_200,
+  });
+
+const Container = styled.div<{ variant: CalloutVariant }>`
+  display: flex;
+  flex-direction: row;
+  border-radius: ${(props) => props.theme.radius(2)}px;
+  align-items: flex-start;
+  background-color: ${(props) => getBackgroundForVariant(props.variant)};
+  padding: ${(props) => props.theme.spacing(8)}px;
+  gap: ${(props) => props.theme.spacing(8)}px;
+`;
+
+const TextContainer = styled.div`
+  display: flex;
+  flex-direction: column;
```

Comment:
I don't think this does anything 👀 This would just behave the same way as a div with no display set, no?

Reply:
Actually, the current styling results in this off-center behaviour:
<img width="346" alt="image" src="https://github.com/secretkeylabs/xverse-web-extension/assets/3322667/f28aac34-661b-4147-8309-d061bcee611e">

You can fix it by keeping these and adding `justify-content: center;` here and changing to `align-items: stretch;` on line 34

Reply:
fixed up styling when titleText and redirectText are both also provided:
 
![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/c2d98037-9b8c-41ae-843f-ba39b8682c6e)

Reply:
Could you try out the changes I noted? It should center the text nicely when it's just a single line of the body with no title and link.

Reply:
you mean vertically centered right? not sure on what the expected behaviour is for single line actually, but zeroheight is all top aligned: 

![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/8db3410e-eeb9-4bcf-ba38-b251db34fd7b)

Reply:
but also the figma has this: 
![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/b262e1c6-f18f-44e5-9c7b-33df76ad4ee3)

Reply:
one line: 
![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/c353c19e-54f7-4f0c-a754-45a12bf22f18)

multi-line: 
![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/cb2a895c-4444-4fcd-9751-825a1e466491)